src/arrayManipulation.hpp

```diff
@@ -298,7 +298,11 @@ void resize( T * const LVARRAY_RESTRICT ptr,
     if( newSize - size > 0 )
     {
       std::size_t const sizeDiff = integerConversion< std::size_t >( newSize - size );
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Warray-bounds"
```

Comment:
does this mean that we won't get any warning whenever trying to access arrays out of bounds?

Reply:
With GCC that will be unfortunately the case.

Reply:
Diagnostic disabling is localized to these specific lines though (where we know it's safe), the rest of the code should be ok

Reply:
This is non-optimal. But I don't see what "array" that is out of bounds in this case. Perhaps something under the covers, but I don't know what "array" would be hanging around that could be out of bounds in a `memset` call. @klevzoff Do you have any ideas?